CMakeLists.txt

```diff
@@ -1,7 +1,16 @@
 cmake_minimum_required(VERSION 3.5)
 project(robin-hood-hashing LANGUAGES CXX)
 
-set(CMAKE_CXX_STANDARD 14)
```

Comment:
Can you update the `.travis.yml` to include  c++11 builds as well?

Reply:
Done